SimpleBlockX4.md

```diff
@@ -0,0 +1,131 @@
+      * To better view where the volumes are within the larger cubes select the wireframe option in the top toolbar. It is directly to the right of the undo arrow on the toolbar.
+      * Most likely the volumes will be slightly off centered in the x direction, but the larger bricks will be centered with one another. This may vary based on the initial CAD program.
+      * To see the blocks better you can click on the axis on the bottom left of the volume viewing area to get a straight on (x,y) view.
+      * To move the blocks in command panel select __Geometry -> Volume -> Transform__ and select __Move__ from the drop down menu. Type `5 6` into the __Volume ID(s)__ box. Then __Select Method -> Distance__ and type 100 into the __X Distance__ box. This distance/direction may vary.
+  4. Hollow out large cubes to match functionality.
+      * In command panel select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. First select volume 6 as volume A and volume 5 as volume B. Check __Keep Originals__ and click __Apply__. You can then delete volume 6 by selecting __Geometry -> Volume -> Delete__ and typing 6 into the __Volume ID(s)__ box and clicking apply.
+      * In command line type `subtract vol 5 from 6 keep` Then type `delete vol 6` to get rid of the original large box.
+      * After these steps you will be left with a large hollow cube that will be used for the graveyard. Next we will create the void volume in much the same way.
+      * In command panel select __Geometry -> Volume -> Boolean__ and select __Subtract__ from the dropdown menu. Select volume 5 as volume A and volumes 1,2,3, and 4 as volume B. Again make sure to check __Keep Originals__ and click __Apply__. Then use the delete action to delete volume 5.
+      * In command line type `subtract vol 1 2 3 4 from 5 keep` or `subtract vol 1 to 4 from 5 keep`. Then type `delete vol 5`.
+      * Now you are left with a brick void volume with cutouts around the inner volumes and a graveyard volume surrounding everything else.
+
+#### Imprinting and Merging Volumes in Cubit/Trelis
+_Imprinting and Merging surfaces accelerates the DAGMC process by combining surfaces and simplifying the process so that no area in space can be owned by more than one volume. The default merge tolerance for distance between surfaces is 5.0e-4._
+
+  1. Imprint and Merge all shared surfaces in the model.
+      * In command panel select __Geometry -> Volume -> Imprint Merge__ and select __Imprint and Merge__ from the dropdown menu. Then type "all" into the __Volume ID(s)__ box and click __Apply__.
+      * In command line type `imprint volume all` and then enter. Then type `merge volume all`. At times it may be helpful to set the merge tolerance to something other than the default, to do this type "merge tol [new merge tolerance]" before running the merge.
+      * Now all surfaces that previously overlapped have been merged into one surface shared by both volumes.

```

Comment:
Rather than "overlapped", the surfaces were "coincident". I tend to think of "overlapping" with respect to volumes and not all overlapping volumes will lead to merged surfaces.